packages/react/src/components/Combobox/ComboboxContext.tsx

```diff
@@ -28,49 +28,49 @@ type ComboboxProvider = {
 const ComboboxContext = createContext<ComboboxContext>({
```

Comment:
Would we not also want `multiselect` inside of the context? See https://github.com/dequelabs/cauldron/blob/158b965d91a283df2647c43d5f52ef89da6297d4/packages/react/src/components/Listbox/ListboxContext.tsx#L14

Reply:
No, it's not needed